sdk/communication/communication-identity/src/models.ts

```diff
@@ -34,7 +34,7 @@ export type CreateUserResponse = WithResponse<CommunicationUserIdentifier>;
 /**
  * Represents the response from issuing a token
  */
-export type IssueTokenResponse = WithResponse<CommunicationUserToken>;
+export type IssueTokenResponse = WithResponse<CommunicationIdentityAccessToken>;
```

Comment:
I think it is better to call it `CommunicationAccessToken` or if in JS name collision is not critical, I would even suggest `AccessToken`

Reply:
I can rename to `CommunicationAccessToken`. In TS you don't really see the name, but I wouldn't want to conflict with `AccessToken`.

Reply:
Cool. Let's 🐹 with `CommunicationAccessToken`